src/segmantic/seg/monai_unet.py

```diff
@@ -548,3 +700,100 @@ def print_table(header, vals, indent="\t"):
                 confusion_metrics,
                 (np.squeeze(x.cpu().numpy()) for x in conf_matrix.aggregate()),
             )
+
+
+def cross_validate(
+    image_dir: Path,
+    labels_dir: Path,
+    tissue_list: Path,
+    output_dir: Path,
+    config_files_dir: Path,
+    checkpoint_file: Path = None,
+    max_epochs: int = 100,
+    generalize_test: bool = False,
+    generalize_img_dir: Optional[Path] = None,
```

Comment:
try `run_monai_cli.py cross-validate --dump-config` to check if the `foo: Optional[Path] = None` behaves the same as the `checkpoint_file: Path = None`.